parm/config/config.ecenfsoi

```diff
@@ -0,0 +1,21 @@
+#!/bin/ksh -x
```

Comment:
What is the purpose of copying `config.ecen` into `config.ecenfsoi` and duplication everything in here?
I also notice that the resources of for `ecen` and `ecenfsoi` are identical.

This comment likely looks relevant to `config.efcsfsoi` and others as well where an `fsoi` version of the config is generated.

Reply:
Where possible, config files for EFSOI tasks source the config files for their respective regular tasks (eg config.ecenfsoi sources config.ecen)